widgetastic_patternfly.py

```diff
@@ -156,6 +156,10 @@ def assert_message(self, text, t=None):
                 e_text = text
             raise AssertionError('assert_message: {}'.format(e_text))
 
+    def assert_success_message(self, text, t=None):
```

Comment:
assert_message() by default have t=None, so i decide to do it same way, so if someone need to add message type